spreadsheets/scalerank/landuse.csv

```diff
@@ -1 +1,72 @@
-kind,scalerank
+kind,operator,zoom::int,area::int,scalerank
+national_park;park or protected land;battlefield,!United States Forest Service,3,>=300000000,1
```

Comment:
1. `park or protected land` is an outmoded v0.10 reference, it should be removed I think?
2. Make sure `!United States Forest Service` matches Olga's change from last week?